pydm/widgets/spinbox.py

```diff
@@ -114,12 +103,11 @@ def value_changed(self, new_val):
         self.setValue(new_val)
         self.valueBeingSet = False
 
-    def send_value(self):
+    def send_value(self, value):
         """
         Method invoked to send the current value on the QDoubleSpinBox to
         the channel using the `send_value_signal`.
         """
-        value = float(self.cleanText())
```

Comment:
I believe that if this line was changed to the following it would probably solve the locale issue:
```python
import locale
value = locale.atof(self.cleanText())
```
Would you mind giving it a try?

Reply:
@hhslepicka I tried a different way to solve this, using the value() method of the QDoubleSpinbox. I think this way is better because we don't need to import another package. Is it ok to do it this way?